k6-operator: polish troubleshooting webpage (#1978)
grafana/k6-docs: docs/sources/k6/next/shared/k6-operator/troubleshooting-how-to.md

   kubectl logs mytest-initializer-xxxxx
   ```

#### `TestRun` with `cleanup` option

> If a `TestRun` has [`spec.cleanup` option](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/usage/executing-k6-scripts-with-testrun-crd/#clean-up-resources) set, as for example [`PrivateLoadZone`](https://grafana.com/docs/grafana-cloud/testing/k6/author-run/private-load-zone/) tests always do, it may be harder to locate and analyze the Pod before it's deleted.

Reviewer: ```suggestion
If a `TestRun` has the [`spec.cleanup` option](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/usage/executing-k6-scripts-with-testrun-crd/#clean-up-resources) set, as [`PrivateLoadZone`](https://grafana.com/docs/grafana-cloud/testing/k6/author-run/private-load-zone/) tests always do, for example, it may be harder to locate and analyze the Pod before it's deleted.
```

In that case, we recommend using observability solutions, like Prometheus and Loki, to store metrics and logs for later analysis.

> As an alternative, it is also possible to watch for the resources manually with the following commands:

Reviewer: ```suggestion
As an alternative, it's also possible to watch for the resources manually with the following commands:
```

If `Stage` is equal to `error`, you can check the logs of k6 Operator.

Conditions can be used as a source of info as well, but it's a more advanced troubleshooting option that should be used if the previous steps weren't enough to diagnose the issue. Note that conditions that start with the `Cloud` prefix only matter in the setting of k6 Cloud test runs, for example, for cloud output and PLZ test runs.

#### Debugging k6 process

> If the script is working locally as expected, and the previous steps show no errors as well, yet you do not see an expected result of a test and suspect k6 process is at fault, you can use the k6 [verbose option](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/k6-options/#options) in the `TestRun` spec:

Reviewer: ```suggestion
If the script is working locally as expected, and the previous steps show no errors as well, yet you don't see an expected result of a test and suspect k6 process is at fault, you can use the k6 [verbose option](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/k6-options/#options) in the `TestRun` spec:
```

Author: Out of curiosity: why the shortened form of the verb is preferable (i.e. `do not` -> `don't`)? It seems like half of my mistakes are because of that :sweat_smile:

Reviewer: Ah, yes, that's a great question! I think there are two main reasons for it. One is that using these contractions can be seen as more conversational or friendly, which is the tone we want for our docs. The second is that if a user is skimming through the docs, they might miss the "not" part, so the contraction can help avoid some confusion.
